tensorflow/python/ops/numpy_ops/np_array_ops_test.py

```diff
@@ -804,6 +804,22 @@ def testSum(self):
   def testAmax(self):
     self._testReduce(np_array_ops.amax, np.amax, 'amax')
 
+  def testSize(self):
+
+    def run_test(arr):
+      onp_arr = arr.numpy() if isinstance(arr, tf.Tensor) else arr
+      self.assertEqual(np_size(arr), onp.size(onp_arr))
```

Comment:
Should this be np_array_ops.size(arr)?

Reply:
Yeah, sorry, `np_size` is my local version when testing.